pom.xml

```diff
@@ -72,17 +72,17 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
-                <version>3.1.0</version>
+                <version>3.2.0</version>
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.21.0</version>
+                <version>3.0.0-M5</version>
```

Comment:
Is it ok to use a non-release version?

Reply:
Normally not, but Surefire has only had milestone releases since more than two years. I hope they finish 3.0.0 soon.